.env.example

```diff
@@ -3,6 +3,11 @@ DEV_MONGO_URI=mongodb://127.0.0.1:27017/booking
 DEV_MONGO_USER=booking-user
 DEV_MONGO_PASS=booking-pass
 
+# MESSAGING Package
```

Comment:
@glrodasz Just to be clear about this, this is for `booking-services`, we are using `fastify` and the `Math` package was created by Santiago as an example, it will be deleted eventually.

Reply:
I know, I shared the Next.js approach as an example of implementation.

Reply:
Oh, ok, sorry, I misunderstood your comment

Reply:
Don't worry, my comment about Next.js wasn't clear enough.